Kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -5,55 +5,36 @@
 
 public class ShapeCollector {
 
-    private List<Shape> figureList = new ArrayList<>();
+    List<Shape> figuresList = new ArrayList<>();
```

Comment:
brakuje teraz private (czemu usunąleś?)
Lepsza nazwa to `shapes`

Reply:
Nie zrobiłem tego celowo. Jednak nadal mam problemy z ogarnięciem gita. Oczywiście poprawiłem.